packages/hub/src/lib/parse-safetensors-metadata.spec.ts

```diff
@@ -83,4 +83,25 @@ describe("parseSafetensorsMetadata", () => {
 		assert.deepStrictEqual(parse.parameterCount, { F32: 859_520_964 });
 		assert.deepStrictEqual(sum(Object.values(parse.parameterCount)), 859_520_964);
 	});
+
+	it("fetch info for sharded (with the default conventional filename) with file path", async () => {
+		const parse = await parseSafetensorsMetadata({
```

Comment:
I noticed that the test is not commit aware, if this model is deleted then this test will fail.

Reply:
yep we can pass a `revision` (and we'd recreate this test if the repo is ever completely deleted)

Reply:
handled in 06ab3c0f33420ddad25974f26d3e15d4521106f6